src/App.test.js

```diff
@@ -1,8 +1,9 @@
+/* eslint-disable jest/expect-expect */
```

Comment:
Maybe we should require this? and that test below should be comparing against a snapshot?

Reply:
Added follow-up issue: https://github.com/canonical-web-and-design/jaas-dashboard/issues/29